### PR TITLE
Move configuration check to django check framework

### PR DESCRIPTION
**Pull request by rsalmaso**
Move all configuration checks from `models/__init__.py` to the Django System check framework.

### CODE REVIEW

File: src/wiki/checks.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    return errors
+
+def check_for_context_processors(app_configs, **kwargs):
+    errors = []
+    context_processors = get_default_engine().context_processors
+    for context_processor in REQUIRED_CONTEXT_PROCESSORS:
+        if context_processor[0] not in context_processors:
+            errors.append(
+                Error(
+                    'needs %s in TEMPLATE[\'OPTIONS\'][\'context_processors\']' % context_processor[0],
```
Comment (atombrella) [inline src/wiki/checks.py]: If you switch to double-quotes, you don't need escape the single-quotes.

File: src/wiki/checks.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    ('sekizai.context_processors.sekizai', 'E009'),
+)
+
+
+def check_for_required_installed_apps(app_configs, **kwargs):
+    errors = []
+    for app in REQUIRED_INSTALLED_APPS:
+        if not apps.is_installed(app[0]):
+            errors.append(
+                Error(
```
Comment (atombrella) [inline src/wiki/checks.py]: I think this would fit on one line, at least the `Error` part.

**Comment by rsalmaso** [comment]
I was pretty sure to have added the tests...

**Comment by atombrella** [comment]
I looked at the test coverage that's added to the PR. LGTM. It's much cleaner than before.

**Comment by benjaoming** [comment]
Seconded, really nice observation and implementation @rsalmaso !

**Pull request merged**